### ISSUE TITLE
Including ensemble forecasts

### ISSUE THREAD

**Issue by HF7weatherman** [orcestra-campaign/weather#20]
If possible, include ensemble forecasts via the ECMWF API. If not, think about switching to ICON forecasts from DWD.

Not urgent yet, though.

**Comment by lkluft** [comment]
The ensemble forecast is also available on the [ECMF open data server](https://data.ecmwf.int/forecasts/20240801/00z/ifs/0p25/enfo/).

However, due to the way the variables are split across the GRIB files, it is rather hard to select only a few variables. As a consequence, one has to download a significant part of the forecast. I think one should only invest time and energy in this, if we really expect a benefit.

**Comment by HF7weatherman** [comment]
@lkluft Could you please let us know once the processing of the exemplary ensemble data as discussed this morning is finished? Thanks! :)

**Comment by lkluft** [comment]
A test dataset can be accessed via
```py
xr.open_zarr("https://swift.dkrz.de/v1/dkrz_948e7d4bbfbb445fbff5315fc433e36a/data_ecmwf/enfo-test.zarr/")
```

The test dataset covers the full forecast period (although the example is quite dated) as well as all ensemble members. To keep disk (or cloud) space reasonable, the dataset is currently limited to `10u`, `10v`, `tcwv`, and `tp`. However, the dataset could be extended to other variables if needed.

**Comment by HF7weatherman** [comment]
@dspraturi @sortega87 @karstenpeters @Lucalino FYI!

UPDATE: It is still the plan to include the ICWV ensemble data (48mm spaghetti plots) into the weather briefing, at least for Barbados as the ITCZ is generally much more diffuse in the Western Atlantic basin. Furthermore, also one spaghetti for the climatological average should be included in that plot (see #22, which is closed now as it will be dealt with in this issue).

**Comment by lkluft** [comment]
At last...

```py
import intake

cat = intake.open_catalog("https://tcodata.mpimet.mpg.de/internal.yaml")
cat.HIFS(datetime="2024-08-30", stream="enfo").to_dask()
```
